Library/MapleBacon/MapleBacon/Storage/InMemoryStorage.swift

```diff
@@ -4,28 +4,54 @@
 
 import UIKit
 
+/**
+ * Handles the storing of images into NSCache
+ */
 public final class InMemoryStorage {
 
+    /// Singleton obj
     public static let sharedStorage = InMemoryStorage()
 
-    private static let DefaultStorageName = "default"
+    /// Default storage name for conveniance init
+    private static let DefaultStorageName = MapleBaconConfig.sharedConfig.storage.defaultStorageName
```

Comment:
As commented above, don't think the singleton config is a good idea. And the way it's used here, won't this be a fixed value at compile time?